bin/selbot2.rb

```diff
@@ -2,6 +2,29 @@
 require 'net/http'
 require 'selbot2'
 require 'octokit'
+require 'faker'
+require "xkcd"
+
+
+def weird
```

Comment:
can you make a new 'weird' lib module instead of just cramming this into selbot2.rb